### DISCUSSION TITLE
Error while Login/register

### DISCUSSION THREAD

**Discussion by Prasanthreddy4a5** [learndevopsonline/learndevopsonline#1705] [Q&A]
Hi,

I have run all the components via terraform and using local_exec provisioner and everything is successful on output screen and  Am able to see the Stans roboshop page from frontend IP . But its giving error while login/register. Please help and let me know if you need anything.
![error](https://github.com/learndevopsonline/learndevopsonline/assets/129600823/9b89d4ee-679f-4caa-ac77-a458b2159297)

**Comment by sreedharm07** [comment]
log in into the component which you are getting error and check status 
systemctl status component name 
if all components are active 
then check all components service files 
after these 2 checks mostly it will work

**Reply by Prasanthreddy4a5** [reply]
Every component I have rerun and checked but still giving the same error, I will log into the component and check the systemctl status and service files and let you know, Thanks.

**Reply by Prasanthreddy4a5** [reply]
Thanks am able to rectify that error, in service files am not giving -dev and now I changed them and its working.


And in shipping I am getting the below error, I logged in the shipping instance and checked the status its fine. But am not getting any country dropdown to select.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129600823/e88d80c0-4c47-4aa7-900c-bbf0763bb49e)

**Reply by sreedharm07** [reply]
check frontend service file and restart frontend once

**Reply by Prasanthreddy4a5** [reply]
Hey, I tried restarting the frontend service file and frontend instance but still am unable to see the country option dropdown

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129600823/ca67845d-8b83-47b3-8b76-2292c5b6eb51)
![1](https://github.com/learndevopsonline/learndevopsonline/assets/129600823/8b5c0e43-d5d6-4353-9b42-7f773e2905ac)

**Reply by sreedharm07** [reply]
check shipping service file 
if you not found any please turn on all instances and give your workstation ip

**Reply by Prasanthreddy4a5** [reply]
Hey @sreedharm07 , Thanks for your help. I again rerun the Terraform  code and now its working fine.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129600823/88b72fa4-1b53-4275-981f-e752a76b403f)
